refactor: :memo: update product descriptions (#399)
seedcase-project/seedcase-project: roadmap/index.qmd

| Status | Software |
|:--------------------------------------------:|:-------------------------|
| {{< var status.done >}} | [`seedcase-sprout`](https://github.com/seedcase-project/seedcase-sprout): Grow organised and FAIR data |
| {{< var status.done >}} | [`check-datapackage`](https://github.com/seedcase-project/check-datapackage): Ensure the compliance of your Data Package metadata |

> | {{< var status.wip >}} | `seedcase-flower`: Turn your metadata into human-readable documentation |

Author: ```suggestion
| {{< var status.wip >}} | [`seedcase-flower`](https://github.com/seedcase-project/seedcase-flower): Turn your metadata into human-readable documentation |
```

| {{< var status.planned >}} | `seedcase-shears`: Processing raw data into a [Tidy](https://vita.had.co.nz/papers/tidy-data.pdf) [Polars DataFrame](https://pola.rs/) |
| {{< var status.planned >}} | `seedcase-propagate`: Submit requests for Data Package data |
| {{< var status.planned >}} | `seedcase-garden`: Track and manage Data Package requests |

: "Wishlist" of software products.

#### Software-related products

| Status | Software-related product |
|:--------------------------------------------:|:-------------------------|
| {{< var status.done >}} | [`template-data-package`](https://github.com/seedcase-project/template-data-package): An opinionated template for building Data Packages using Seedcase |

> | {{< var status.done >}} | [`template-python-package`](https://github.com/seedcase-project/template-python-package): A template repository to use when making a Python package for Seedcase |

Author: ```suggestion
| {{< var status.done >}} | [`template-python-package`](https://github.com/seedcase-project/template-python-package): An opinionated toolkit for building Python packages |
```

> | {{< var status.done >}} | [`template-website`](https://github.com/seedcase-project/template-website): An opinionated template for making Quarto-based websites |

Author: ```suggestion
| {{< var status.done >}} | [`template-website`](https://github.com/seedcase-project/template-website): Set up and manage website projects |
```

| Status | Guides or manuals |
|:--------------------------------------------:|:-------------------------|
| {{< var status.done >}} | [`seedcase-sprout`](https://sprout.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-sprout` |
| {{< var status.done >}} | [`check-datapackage`](https://check-datapackage.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-check-datapackage` |
| {{< var status.done >}} | [`template-data-package`](https://template-data-package.seedcase-project.org/docs/guide) usage guide: Using the template to make data packages |
| {{< var status.done >}} | [`template-website`](https://template-website.seedcase-project.org/docs/guide) usage guide: Using the template to make websites |
| {{< var status.done >}} | [`template-python-package`](https://template-python-package.seedcase-project.org/docs/guide) usage guide: Using the template to make Python packages |

> | {{< var status.wip >}} | [`guidebook`](https://guidebook.seedcase-project.org/): Guidebook on collaborating as a team on projects kept on GitHub |

Author: ```suggestion
| {{< var status.wip >}} | [`guidebook`](https://guidebook.seedcase-project.org/): A guidebook for team collaboration on GitHub |
```

| Status | Information resource |
|:--------------------------------------------:|:-------------------------|
| {{< var status.done >}} | [`seedcase-sprout`](https://sprout.seedcase-project.org/docs/reference/): Reference docs for Sprout |
| {{< var status.done >}} | [`check-datapackage`](https://check-datapackage.seedcase-project.org/docs/reference/): Reference docs for `check-datapackage` |

> | {{< var status.ongoing >}} | [`design`](https://github.com/seedcase-project/design): General design principles and patterns for the organisation, teams, software, and data used throughout the Seedcase Project |

Author: ```suggestion
| {{< var status.ongoing >}} | [`design`](https://github.com/seedcase-project/design): Principles and patterns for development in the Seedcase Project |
```

> | {{< var status.ongoing >}} | [`team`](https://github.com/seedcase-project/team): Documentation specific to the Seedcase team, like onboarding, common configuration files, and meeting agendas and minutes |

Author: ```suggestion
| {{< var status.ongoing >}} | [`team`](https://github.com/seedcase-project/team): Internal docs for onboarding and admin information |
```

| Status | Deliverable |
|:--------------------------------------------:|:-------------------------|

> | {{< var status.ongoing >}} | [`community`](https://github.com/seedcase-project/community): Content for community building, outreach, and news about the Seedcase Project |

Author: ```suggestion
| {{< var status.ongoing >}} | [`community`](https://github.com/seedcase-project/community): Community building, outreach, and updates on the Seedcase Project |
```

> | {{< var status.ongoing >}} | [`decisions`](https://github.com/seedcase-project/decisions): Archival records of decisions made about tools or processes we use in the Seedcase Project |

Author: ```suggestion
| {{< var status.ongoing >}} | [`decisions`](https://github.com/seedcase-project/decisions): A record of decisions on tools and processes in the Seedcase Project |
```
